js/functions.ts

```diff
@@ -15,4 +15,42 @@ const getNumbersFromString = (input: number | string) => {
 	return parseInt(stringWithOnlyDigits, 10);
 };
 
-export { isShorterThan, isPalindrome, getNumbersFromString };
+const parseTimeToNumbers = (time: string) => {
+	const result: number[] = [];
+
+	time.split(':').forEach((timePiece) => {
+		result.push(Number(timePiece));
+	});
+
+	return result;
```

Comment:
map тут будет короче
```suggestion
	const result = time.split(':').map(Number)
	
	return result;
```